jwcrypto/jwa.py

```diff
@@ -28,6 +28,12 @@
 from jwcrypto.common import json_decode
 from jwcrypto.jwk import JWK
 
+__all__ = [
+    'JWA',
+    'JWAAlgorithm',
```

Comment:
JWAAlgorithm is not really public ...

Reply:
What for? `JWAAlgorithm` is an abstract class that allows you to write your own algorithm, defining which methods should be implemented.

Reply:
It is available but should be imported explicitly imo, it is extremely rare (and somewhat discouraged) to implement your own crypto primitieves, and if a new RFC that defines new algorithms come out I rather have the implementation directly in jwcrypto itself.

Reply:
I agree. I will remove `JWAAlgorithm`